c/tskit/tables.h

```diff
@@ -961,19 +961,61 @@ on and may change arbitrarily between versions.
 */
 void tsk_individual_table_print_state(const tsk_individual_table_t *self, FILE *out);
 
-/** @} */
+/**
+@brief Replace this table's data by copying from a set of column arrays
 
-/* Undocumented methods */
+@rst
+Clears the data columns of this table and then copies column data from the specified
+set of arrays. The supplied arrays should all contain data on the same number of rows.
+The metadata schema is not affected.
```

Comment:
Should we say here somewhere that if there's no locations (for instance) to pass `NULL, 0`? Or maybe that's general-purpose advice about ragged columns we don't need to repeat.